server/src/utils/token.util.ts

```
@@ -0,0 +1,10 @@
+import crypto from 'crypto';
+
+/**
+ * Generates a secure random token.
+ * @param length - The length of the token (default is 32).
+ * @returns A secure random token as a string.
+ */
+export const generateToken = (length: number = 32): string => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)